src/transaction/log_recovery.c

```diff
@@ -1172,23 +1162,34 @@ log_recovery_redo (THREAD_ENTRY * thread_p, log_recovery_context & context)
 	      }
 	  }
 
+	  /* Read the log record information from log and then if redo is required apply it */
+
+	  const LOG_LSA rcv_lsa = redo_context.m_reader.get_lsa ();	/* Address of redo log record */
+	  /* skip log record header HEADER */
+	  redo_context.m_reader.add_align (sizeof (LOG_RECORD_HEADER));
```

Comment:
`add_align` was used conditionally for COMMIT/ABORT branch. Is moving this above the conditional scope safe?